lib/seeing_is_believing/binary/config.rb

```diff
@@ -262,7 +270,8 @@ def finalize(stdin, stdout, stderr, file_class)
           add_error("Cannot give a program body and a filename to get the program body from.")
         elsif filename && file_class.exist?(filename)
           self.lib_options.stdin = stdin
-          self.body = file_class.read filename
+          # TODO: this doesn't seem like a safe assumption
+          self.body = file_class.read filename, external_encoding: "utf-8"
```

Comment:
For the larger background on encodings in Windows, see the blog post. On a case-by-case basis, a lot of the choices I made fall into the category of makes-the-test-pass-but-might-not-be-the-right-choice :-/